components/Common/Search/States/WithSearchBox.tsx

```diff
@@ -188,7 +182,10 @@ export const WithSearchBox: FC<SearchBoxProps> = ({ onClose }) => {
                 role="combobox"
                 type="search"
                 className={styles.searchBoxInput}
-                onChange={event => setSearchTerm(event.target.value)}
+                onChange={event => {
+                  setSearchTerm(event.target.value);
+                  search(event.target.value, selectedFacet);
```

Comment:
i know this PR has went through a fair amount of discussion, but without commenting on any of that, I'd say having this change handler perform two operations like this is not more clear to me. The PR is now open for over a month, and I'd consider that ample time for others to weigh in.

I am closing.

I hope that you can find other areas of the site or node.js project to help out with in the future.